stdlib/public/Cxx/cxxshim/libcxxstdlibshim.h

```diff
@@ -4,19 +4,19 @@
 
 /// Used for std::string conformance to Swift.Hashable
 typedef std::hash<std::string> __swift_interopHashOfString;
-inline std::size_t __swift_interopComputeHashOfString(std::string str) {
+inline std::size_t __swift_interopComputeHashOfString(const std::string &str) {
```

Comment:
Hmm, I guess making the parameter a const reference avoids the IR verifier failure?

Reply:
Yep!